docs/_assets/attributes.adoc

```diff
@@ -16,6 +16,7 @@
 //Latest version
 :ProductVersion: 0.6
 :tag: v0.6.0
```

Comment:
I noticed these "stale" versions and started wondering if they should be 0.7 and 0.7.0-SNAPSHOT so I raised https://github.com/kroxylicious/kroxylicious/pull/1443